packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.tsx

```diff
@@ -82,33 +79,43 @@ const ConfirmResetPassword = (_: RouteComponentProps) => {
     instructionText: `Click the link emailed to ${email} within the next hour to create a new password.`,
   };
 
-  return (
-    <AppLayout>
-      {passwordResetResend && (
-        <Banner
-          type={BannerType.success}
-          dismissible
-          setIsVisible={setPasswordResetResend}
-        >
-          <FtlMsg
-            id="resend-pw-reset-banner"
-            vars={{ accountsEmail: FIREFOX_NOREPLY_EMAIL }}
+  const navigateToPasswordReset = useCallback(() => {
+    navigate('reset_password?showReactApp=true', { replace: true });
```

Comment:
What would you think of adding a TODO to remove this flag query param once everything is fully deployed?

Reply:
I'll add this in the WIP branch mentioned above too - since it's optional for this PR and tests are already passing :)

Reply:
Actually, I don't think we need this feature flag here since we're staying within the react app (and no other calls to navigate include it). I'll go ahead and remove it in that other WIP branch.